### ISSUE TITLE
bundler tries to download old version of has_friendship gem

### ISSUE THREAD

**Issue by cdrainxv** [sungwoncho/has_friendship#61]
I keep getting this error:

```
Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    rails (~> 5.2.1)

    has_friendship was resolved to 0.0.3, which depends on
      rails (~> 4.1.8)
```
The error could be on my part because the gemspec seems to support rails up to 5.3.0.

EDIT: 
I took a look at rubygems.org and noticed the version here and there are not consistent. I then attempted to state that version in my gemfile, but alas the gem depends on rails < 5.2.0 while I am on 5.2.1. 

I noticed that there was discussion about the owner not being able to maintain this repo and a @chevinbrown has volunteered to be a maintainer, so I'll just await the decision then. For now a simple solution is to install the gem using this git source as at least it's compatible with rails up to 5.3.0:

`gem 'has_friendship', git: 'git@github.com:sungwoncho/has_friendship.git'`

**Comment by chevinbrown** [comment]
@cdrainxv you're correct--the source is updated, but we've not had a release. I had offered to help maintain, but I am not the gem owner. Unless @sungwoncho is willing to share that ownership, there's not much more I can do. 

For now, the best thing to do would be what you've recommended. I would recommend you just lock to the commit sha `ref: '857b36a'`.

I don't mean to disparage at all, but I'll probably bow out of maintainership...I had hoped to support it's growth but given we can't get a release out, I'll probably focus my efforts elsewhere. 

❤️

**Comment by sungwoncho** [comment]
@chevinbrown If you are still interested in an ownership, please share your email here or on my email and I will add you as a owner on rubygems. Sorry about the coordination.

**Comment by chevinbrown** [comment]
@sungwoncho `chevinbrown@gmail.com`. 
I won't have _a lot_ of time to contribute, but I would like to help OSS and move this gem forward a tad. 😄

**Comment by chevinbrown** [comment]
@cdrainxv just released `1.1.2` 🎉
